client/my-sites/site-settings/form-security.jsx

```diff
@@ -119,6 +125,13 @@ const connectComponent = connect( ( state ) => {
 		'akismet'
 	);
 
+	const hasAkismetFeature = siteHasFeature( state, siteId, WPCOM_FEATURES_AKISMET );
+	const hasAntiSpamFeature = siteHasFeature( state, siteId, WPCOM_FEATURES_ANTISPAM );
+	const hasJetpackAntiSpamProduct = getSiteProducts( state, siteId )?.some( isJetpackAntiSpam );
+
+	const includeAkismetKeyField =
+		! isAtomic && ( hasAkismetFeature || hasAntiSpamFeature || hasJetpackAntiSpamProduct );
```

Comment:
This maybe deserves a comment that this condition, including the `settings.akismet` in `getFormSettings`, is exactly identical to the one in `SpamFilteringSettings` that decides whether the UI for the Akismet key is disabled.